assets/src/edit-story/components/inspector/inspectorLayout.js

```diff
@@ -60,6 +60,7 @@ function InspectorLayout() {
   return (
     <Layout ref={inspector}>
       <TabView
+        label={__('Inspector selection', 'web-stories')}
```

Comment:
```suggestion
        label={__('Inspector Selection', 'web-stories')}
```

For consistency with `Element Library Selection`